XIVSlothCombo/Combos/PvE/PLD.cs

```diff
@@ -382,13 +345,16 @@ protected override uint Invoke(uint actionID, uint lastComboMove, float comboTim
             {
                 if (actionID is SpiritsWithin or CircleOfScorn)
                 {
-                    if (level is >= Levels.SpiritsWithin and <= Levels.Expiacion)
+                    int spirit = GetLevel(SpiritsWithin);
+                    int expiaction = GetLevel(Expiacion);
+
+                    if (level >= spirit && level <= expiaction)
                         return CalcBestAction(actionID, SpiritsWithin, CircleOfScorn);
 
-                    if (level >= Levels.Expiacion)
+                    if (level >= expiaction)
```

Comment:
This is spelled wrong so probably isn't working unless we've got it wrong elsewhere - might be worth just removing PLD stuff from this PR in anticipation of the rework tbh

Reply:
PLD Removed